.hlint.yaml

```diff
@@ -39,18 +29,65 @@
 - warn: {lhs: Control.Exception.evaluate, rhs: evaluateWHNF}
 - warn: {lhs: Control.Exception.evaluate (force x), rhs: evaluateNF x}
 - warn: {lhs: Control.Exception.evaluate (x `deepseq` ()), rhs: evaluateNF_ x}
-
 - warn: {lhs: void (evaluateWHNF x), rhs: evaluateWHNF_ x}
 - warn: {lhs: void (evaluateNF x), rhs: evaluateNF_ x}
 
-- hint: {lhs: "foldl' (flip f)", rhs: "flipfoldl' f"}
+## Monad.Trans
+
+- warn: {lhs: flip runReaderT r t, rhs: usingReaderT r t}
+- warn: {lhs: flip runReader r t, rhs: usingReader r t}
+- warn: {lhs: flip runStateT r t, rhs: usingStateT r t}
+- warn: {lhs: flip runState r t, rhs: usingState r t}
+- hint: {lhs: fst <$> usingStateT s st, rhs: evaluatingStateT s st}
+- hint: {lhs: fst (usingState s st), rhs: evaluatingState s st}
+- hint: {lhs: snd <$> usingStateT s st, rhs: executingStateT s st}
+- hint: {lhs: snd (usingState s st), rhs: executingState s st}
+
+## Containers
+- hint: {lhs: Data.HashMap.Lazy.keys, rhs: Universum.keys}
+- hint: {lhs: Data.HashMap.Strict.keys, rhs: Universum.keys}
+- hint: {lhs: Data.Map.Lazy.keys, rhs: Universum.keys}
+- hint: {lhs: Data.Map.Strict.keys, rhs: Universum.keys}
+- hint: {lhs: Data.IntMap.Lazy.keys, rhs: Universum.keys}
+- hint: {lhs: Data.IntMap.Strict.keys, rhs: Universum.keys}
+
+- hint: {lhs: Data.HashMap.Lazy.elems, rhs: Universum.elems}
+- hint: {lhs: Data.HashMap.Strict.elems, rhs: Universum.elems}
+- hint: {lhs: Data.Map.Lazy.elems, rhs: Universum.elems}
+- hint: {lhs: Data.Map.Strict.elems, rhs: Universum.elems}
+- hint: {lhs: Data.IntMap.Lazy.elems, rhs: Universum.elems}
+- hint: {lhs: Data.IntMap.Strict.elems, rhs: Universum.elems}
+
+- hint: {lhs: Data.HashMap.Lazy.toList, rhs: Universum.toPairs}
+- hint: {lhs: Data.HashMap.Strict.toList, rhs: Universum.toPairs}
+- hint: {lhs: Data.Map.Lazy.toList, rhs: Universum.toPairs}
+- hint: {lhs: Data.Map.Lazy.assocs, rhs: Universum.toPairs}
+- hint: {lhs: Data.Map.Strict.toList, rhs: Universum.toPairs}
+- hint: {lhs: Data.Map.Strict.assocs, rhs: Universum.toPairs}
+- hint: {lhs: Data.IntMap.Lazy.toList, rhs: Universum.toPairs}
+- hint: {lhs: Data.IntMap.Lazy.assocs, rhs: Universum.toPairs}
+- hint: {lhs: Data.IntMap.Strict.toList, rhs: Universum.toPairs}
+- hint: {lhs: Data.IntMap.Strict.assocs, rhs: Universum.toPairs}
+
+- warn: { lhs: Data.Map.toAscList (Data.Map.fromList x)
+        , rhs: Universum.sortWith fst x
+        }
+- warn: { lhs: Data.Map.toDescList (Data.Map.fromList x)
+        , rhs: Universum.sortWith (Down . fst) x
+        }
+
+- warn: {lhs: Data.Set.toList (Data.Set.fromList l), rhs: Universum.sortNub}
```

Comment:
Because `lhs` rule has `l` (`Data.Set.toList (Data.Set.fromList l)`) you need to use `l` in `rhs` as well, like `sortNub l`. This might be important because there's tool to automatically apply `hlint` suggestions and we don't want to do invalid transformations.